src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -605,6 +627,27 @@ private PrecompileArguments GetPrecompileArgumentsFor(ITaskItem assemblyItem, st
             aotArgs.Add($"static");
         }
 
+        if (UseDirectPInvoke)
+        {
+            aotArgs.Add($"direct-pinvoke");
+        }
+
+        if (DirectPInvokes.Length > 0)
+        {
+            foreach (var directPInvoke in DirectPInvokes)
+            {
+                aotArgs.Add($"direct-pinvokes={directPInvoke}");
+            }
+        }
+
+        if (DirectPInvokeLists.Length > 0)
+        {
+            foreach (var directPInvokeList in DirectPInvokeLists)
+            {
+                aotArgs.Add($"direct-pinvoke-lists={directPInvokeList}");
```

Comment:
See above, but maybe this could argument should just be named direct-pinvoke-list to indicate that it only accept one value.

The argument to the task could still be named DirectPInvokeLists since it will be mapped to an item group in msbuild.

Reply:
Modified to also accept multiple values akin to https://github.com/dotnet/runtime/pull/79721#discussion_r1067362499